pennylane/_qutrit_device.py

```diff
@@ -80,9 +87,64 @@ def capabilities(cls):
         capabilities.update(model="qutrit")
         return capabilities
 
+    # Added here for tests to pass (See tests.test_qutrit_device.ExtractStatistics.test_results_no_state())
+    # The test deleted the state property of QutritDevice to test if the appropriate error is raised by
+    # `QutritDevice.statistics()` when requesting the state as a return type, but was failing as the test wouldn't
+    # actually delete the state attribute, as it was only explicitly defined in `QubitDevice`.
+    @property
+    def state(self):
+        """Returns the state vector of the circuit prior to measurement.
+
+        .. note::
+
+            Only state vector simulators support this property. Please see the
+            plugin documentation for more details.
+        """
+        raise NotImplementedError
```

Comment:
That's correct. I totally missed that while trying to debug this issue. Thanks for pointing it out. I will note though that it seems redundant to write a test where I delete `QubitDevice.state` rather than `QutritDevice.state` (which I can't delete because I will get an `AttributeError`) in a test file meant to test `QutritDevice`, and I am starting to wonder if the test case is needed at all or not once I remove the redundant `QutritDevice.state` attribute. Let me know what you think :)

Reply:
I think it can still be a valid test case: it tests that the same behaviour is observed with `QutritDevice` as with `QubitDevice` because of inheritance. Likely not a super crucial one, but can still have it.